src/components/Navbar.tsx

```diff
@@ -150,25 +168,28 @@ export default function Navbar() {
           lg:left-0 lg:flex-row lg:items-center lg:px-8 lg:pt-0
         ` + (open ? "left-0" : "-left-full")
         }>
-        <div className="px-14 pb-0 lg:px-0 lg:pb-0 flex flex-col lg:flex-row ">
-          <a
-            className="sr-only focus:not-sr-only"
-            href="#main-content"
-          >{t.toContent}</a>
-          <NavLink class="px-0 lg:px-4 p-4" href="/">{t.home}</NavLink>
-          <Group links={[
-            { href: "/förföretag", text: t.forCompanies },
-            { href: "/faq", text: "faq" },
-          ]} />
-          <NavLink class="mb-4 lg:hidden px-0 lg:px-4" href="/kontakt">{t.contact}</NavLink>
-          {/*<NavLink class="px-14 lg:px-0" href="/förstudenter">{t.forStudents}</NavLink>*/}
-          {/*<NavLink class="px-14 lg:px-0" href="/mässan">{t.about}</NavLink>*/}
+          <div className="px-14 pb-0 lg:px-0 lg:pb-0 flex flex-col lg:flex-row ">
+            <a
+              className="sr-only focus:not-sr-only"
+              href="#main-content"
+            >{t.toContent}</a>
+            <NavLink class="px-0 lg:px-4 p-4" href="/">{t.home}</NavLink>
+            <Group links={[
+              { href: "/förföretag", text: t.forCompanies },
+              { href: "/faq", text: "faq" },
+              // isLoggedIn.data == true ?
+              //   { href: "/utställare", text: t.exhibitorSettings } :
+              //   { href: "/logga-in", text: t.login },
+            ]} />
+            <NavLink class="mb-4 lg:hidden px-0 lg:px-4" href="/kontakt">{t.contact}</NavLink>
+            {/*<NavLink class="px-14 lg:px-0" href="/förstudenter">{t.forStudents}</NavLink>*/}
+            {/*<NavLink class="px-14 lg:px-0" href="/mässan">{t.about}</NavLink>*/}
```

Comment:
The space between "Home" and "For Companies" is greater than before. Is this intentional?

Reply:
No, i misread a merge conflict and accidentally duplicated some code. Fixed now!